qiskit/visualization/pulse/matplotlib.py

```diff
@@ -224,19 +224,24 @@ def __init__(self, style):
         """
         self.style = style or PulseStyle()
 
-    def draw(self, pulse, dt, interp_method, scaling=1):
+    def draw(self, pulse, dt, interp_method, scale=1, scaling=1):
```

Comment:
It looks like this should be:
```suggestion
    def draw(self, pulse, dt, interp_method, scale=1, scaling=None):
```
so that we don't hit the warning by default

Reply:
Oops, missed that!
Thank you.